test/parallel/test-permission-inspector-brk.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const { spawnSync } = require('child_process');
+const fixtures = require('../common/fixtures');
+const file = fixtures.path('permission', 'inspector-brk.js');
+
+common.skipIfWorker();
+common.skipIfInspectorDisabled();
+
+const { status, stderr } = spawnSync(
+  process.execPath,
+  [
+    '--experimental-permission',
+    '--allow-fs-read=*',
```

Comment:
I see `node --experimental-permission --inspect-brk --eval "console.log('Hi')"` as minimal command line. No need for the --allow-fs-read.